### PR TITLE
Enable network manager in engine

### PR DESCRIPTION
**Pull request by eslutsky**
Network scripts are deprecated Since el8

### CODE REVIEW

**Comment by eslutsky** [comment]
its verified

File: tasks/bootstrap_local_vm/02_create_local_vm.yml [hunk 1]
```diff
@@ -105,12 +105,13 @@
...
           - {src: templates/meta-data.j2, dest: "{{ he_local_vm_dir }}/meta-data"}
-          - {src: templates/network-config-dhcp6.j2, dest: "{{ he_local_vm_dir }}/network-config"}
+          - {src: templates/network-config-dhcp.j2, dest: "{{ he_local_vm_dir }}/network-config"}
       - name: Create ISO disk
         command: >-
           mkisofs -output {{ he_local_vm_dir }}/seed.iso -volid cidata -joliet -rock -input-charset utf-8
           {{ he_local_vm_dir }}/meta-data {{ he_local_vm_dir }}/user-data
-          {{ he_local_vm_dir + '/network-config' if ipv6_deployment else None }}
+          {{ he_local_vm_dir }}/network-config
+
```
Comment (arachmani) [inline tasks/bootstrap_local_vm/02_create_local_vm.yml]: remove new line

File: templates/network-config-dhcp.j2 [hunk 2]
```diff
@@ -4,4 +4,9 @@ config:
     name: eth0
     mac_address: {{ he_vm_mac_addr }}
     subnets:
+{% if ipv6_deployment %}
       - type: dhcp6
+{% else %}
+      - type: dhcp
+{% endif %}
+
```
Comment (arachmani) [inline templates/network-config-dhcp.j2]: remove new line

**Comment by arachmani** [comment]
verified on el8

**Comment by didib** [comment]
ci merge please

**Pull request merged**